lmdeploy/messages.py

```diff
@@ -196,6 +196,7 @@ class TurbomindEngineConfig:
     cache_chunk_size: int = -1
     cache_block_seq_len: int = 64
     enable_prefix_caching: bool = False
+    enable_ep: bool = False
```

Comment:
add a new parameter `ep` instead of `enable_ep`. 
EP and TP are independent of each other